api/XMLHttpRequestEventTarget.json

```diff
@@ -61,7 +61,7 @@
               "version_added": null
             },
             "edge": {
-              "version_added": null
+              "version_added": "18"
```

Comment:
This is a tricky one. The `XMLHttpRequestEventTarget` interfaces was exposed first in Edge 18, but that's not what matters. I'll revert these changes and add them to https://github.com/mdn/browser-compat-data/issues/3463.